packages/bot-web-ui/src/components/dashboard/tour-guide.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { localize } from '@deriv/translations';
+import { storeSetting, getSetting } from '../../utils/settings';
+import { Text } from '@deriv/components';
+import { getUrlBase } from '@deriv/shared';
+import RootStore from 'Stores/index';
+import { connect } from 'Stores/connect';
+
+type TTourGuide = {
+    Label: string | boolean;
+    Content: string[];
+    img?: string;
+    className: string;
+    setJoyrideStepIndex: number;
+    setActiveTab: (param: number) => void;
+};
+
+const TourGuide = ({ Label, Content, img, className, setJoyrideStepIndex, setActiveTab }: TTourGuide) => {
+    React.useEffect(() => {
+        setActiveTab(setJoyrideStepIndex);
+    }, [setJoyrideStepIndex]);
+    return (
+        <React.Fragment>
+            <div>
+                <Text as='h' align='left' size='s' line_height='l' weight='bold' className='intial_bot_tour'>
```

Comment:
default align is left and default size is `s` we can remove them

Reply:
done ✅